scripts/sample-reads-randomly.py

```diff
@@ -162,8 +163,8 @@ def main():
     if len(reads) == 1:
         print('Writing %d sequences to %s' %
               (len(reads[0]), output_filename), file=sys.stderr)
-        if not output_file:
-            output_file = open(output_filename, 'w')
+        output_file = get_file_writer(open(output_filename, 'w'), args.gzip,
```

Comment:
I usually put a newline after the first open parenthesis to make the long lines like this wrap better